python/src/RandomVectorMetropolisHastings_doc.i.in

```diff
@@ -31,9 +31,9 @@ Examples
 
 Sample from a random variable:
 
->>> source = ot.RandomVector(ot.Normal())
+>>> randomVector = ot.RandomVector(ot.Normal())
 >>> initialState = [3.0]
->>> sampler = ot.RandomVectorMetropolisHastings(source, initialState)
+>>> sampler = ot.RandomVectorMetropolisHastings(randomVector, initialState)
```

Comment:
I am not sure we need an example that trivial: `sampler.getRealization()` is here equivalent to `randomVector.getRealization()`!